PowerEditor/installer/nativeLang/french.xml

```diff
@@ -1068,26 +1072,26 @@ Translation note:
 
 				<NewDoc title="Nouveau document">
 					<Item id="6401" name="Format (fin de ligne)"/>
-					<Item id="6402" name="Windows (CR LF)"/>
+					<Item id="6402" name="Windows™ (CR LF)"/>
```

Comment:
If we put `™` on Windows, I think we should do the same for Unix & Macintosh (the following 2 lines).